website/blog/2022-12-2-your-own-backups.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: your-own-backups
+title: "Owning Your Identity and Data For The Price of One ☕️/Month "
+authors: nica
+tags: [corso, microsoft 365]
+---
+
+
+I recently saw a tweet that really spoke to me:
+
+![Gian Paolo tweet stating The only long-term solution to maintain your identity is: 1. your own domain name 2. your own website/blog 3. several backups](../static/img/blog/gian_tweet_backups.png)
+
```

Comment:
Like the first blog post, please add a 
```
<!-- truncate -->
```

Here or wherever you want the preview to end?